resources/migrations/056_update_migrations.yaml

```diff
@@ -169,7 +169,320 @@ databaseChangeLog:
                   type: ${text.type}
                   constraints:
                     nullable: false
-# >>>>>>>>>> DO NOT ADD NEW MIGRATIONS BELOW THIS LINE! ADD THEM ABOVE <<<<<<<<<<
+
+  - changeSet:
+      id: v56.2025-07-03T16:00:00
+      author: qnkhuat
+      comment: Create data_app table for data apps functionality
+      preConditions:
+        - onFail: MARK_RAN
+        - not:
+            - tableExists:
+                tableName: data_app
+      changes:
+        - createTable:
+            tableName: data_app
+            remarks: Applications
+            columns:
+              - column:
+                  name: id
+                  remarks: Unique ID
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  remarks: App name
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+              - column:
+                  name: url
+                  remarks: unique url of an app
+                  type: varchar(254)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: status
+                  remarks: Status of the app (private, published, archived)
+                  type: varchar(32)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: entity_id
+                  remarks: Entity id
+                  type: char(21)
+                  constraints:
+                    nullable: true
+              - column:
+                  remarks: The timestamp of when the app was created
+                  name: created_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: The timestamp of when the app was updated
+                  name: updated_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+      rollback:
+        - dropTable:
+            tableName: data_app
+
+  - changeSet:
+      id: v56.2025-07-03T16:01:00
+      author: qnkhuat
+      comment: Create data_app_definition table for data apps functionality
```

Comment:
very nit: Sanya and I have been calling this `configuration` rather than `definition`. A problem with both names is that they sound like the `name` and `url` would be included though. 

Let's leave it like this for now, maybe a better name will occur to us in the future.